XOSANSR.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/python
+#
+#
+# XOSANSR: Xen Orchestra SAN filesystem based storage repository
```

Comment:
I think we should make some modifications here:

* we are introducing a generic Gluster driver so the name should be related to that (Gluster driver)
* this code is Open Source unlike what's in this header

Reply:
Ok, let's keep common header for both SR drivers. @nraynaud Can you do the modifications?